tests/src/Kernel/Entity/GetBundleByBundleTest.php

```diff
@@ -0,0 +1,557 @@
+<?php
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\block_content\Entity\BlockContentType;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelper;
+
+/**
+ * Tests retrieving group (content) bundles by group (content) bundles.
```

Comment:
if not too long, maybe `Tests retrieving group / group content bundles by group / group content bundles.`